_episodes/14.md

```diff
@@ -0,0 +1,330 @@
+For instance, if you create a table with the Iceberg Java API, [it creates
+Iceberg tables with <table_type, ICEBERG>](https://github.com/apache/iceberg/blob/996ed979f396f2c7cc12ca824a3fe758f2c486ce/hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java#L222)
+but Trino [creates and reads with <table_type, iceberg>](https://github.com/prestosql/presto/blob/master/presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/HiveTableOperations.java#L190).
+See [Issue 1592](https://github.com/trinodb/trino/issues/1592) for status and 
+details. In general, we can track some of the broader changes that are being 
+made to [the Iceberg connector here](https://github.com/trinodb/trino/issues/1324).
+
+## PR of the week: PR 1067 Add Iceberg connector
+
+A huge shoutout goes to [Parth Brahmbhatt](https://github.com/Parth-Brahmbhatt),
+a Senior Software Engineer at Netflix who created this weeks' 
+[PR of the week](https://github.com/trinodb/trino/pull/1067).
+
+[Release 318](/docs/current/release/release-318.html), 
+introduced this code that supported querying tables from Apache Iceberg in
+Trino. While the code existed, the Iceberg connector code wasn't officially
+released or documented until a little over a year later in 
+[release 341](/docs/current/release/release-341.html) once the connector reached
+maturity. 
+
+## Demo: Creating tables through Iceberg and reading them through Trino
+
+For this weeks' demo, I wanted to play around with the Iceberg Java API directly.
+You also have the option to use Trino, Spark, or other to ingest and query the
+data, but I wanted to use vanilla Iceberg API's to experience the API and
+hopefully solidify my learning of Iceberg concepts in the process. Make sure you
+follow the instructions in the repository if you don't have Docker or Java
+installed.
+
+Let's start up a local Trino coordinator and hive metastore. Clone this 
+repository and navigate to the `iceberg/trino-iceberg-minio` directory. Then
+start up the containers using Docker Compose.
+
+```
+git clone git@github.com:bitsondatadev/trino-getting-started.git
+
+cd iceberg/trino-iceberg-minio
+
+docker-compose up -d
+```
+
+In your favorite IDE, open the files under `iceberg/iceberg-java` into your
+project and run the `IcebergMain` class.
+
+This class creates a logging table if it doesn't exist. Once you run this code,
+you can check to see that the table in Trino exists in the metastore under
+TABLE_PARAMS. But, if run `SHOW TABLES IN iceberg.logging;` you'll notice that
+the table doesn't show up due to [the issue we discussed above](https://github.com/trinodb/trino/issues/1592).
+
+Let's update the TABLE_PARAMS entry in the metastore db and then query the table
+again.
+
+## Question of the week: Why do we still depend on the Hive metastore if metadata for Iceberg saves to the filesystem?
+
+We kept the metastore as many tests run around using the metastore that exist
```

Comment:
Who is "we" in this context .. I think this should be changed to either 

The Iceberg community

or 

The Trino community 

or whatever is correct.

Reply:
maybe just Trino as a system

Reply:
I am not sure what is true..